version/news/NEWS-2023.09.1-desert-sunflower.md

```diff
@@ -2,6 +2,7 @@
 
 ### New
 #### RStudio
+- GitHub Copilot is available as an opt-in integration with RStudio. This feature is currently in public Preview, and has only been tested on RStudio desktop. Copilot support is expected to be generally available in RStudio desktop, RStudio server and Posit Workbench in a future release.
```

Comment:
Not sure if the capitalization is needed, but just in case:

```suggestion
- GitHub Copilot is available as an opt-in integration with RStudio. This feature is currently in public Preview, and has only been tested on RStudio Desktop. Copilot support is expected to be generally available in RStudio Desktop, RStudio Server and Posit Workbench in a future release.
```

Reply:
Capitalization makes sense. Thanks!